### PR TITLE
[UI] Payment Triggers UI Improvements

### PR DESCRIPTION
**Pull request by johnkuney**
Related to #

Description
---
Some style updates for the payments triggers section as discussed 

Test plan
---
Preview the site and check everything looks good and functions well

### CODE REVIEW

File: components/Paybutton/PaybuttonTrigger.tsx [hunk 1]
```diff
@@ -147,110 +151,218 @@ export default ({ paybuttonId, emailCredits }: IProps): JSX.Element => {
...
               </div>
-            </div>
             </form>
-          <form onSubmit={(e) => { void handleSubmitEmailTrigger(getSubmitTriggerHandler('email'))(e) }} method='post'>
-            <div>
+          </div>
+
+          <div className={style.form_ctn}>
+            <div className={style.trigger_header}>
               <h5>Receive Email</h5>
```
Comment (Klakurka) [inline components/Paybutton/PaybuttonTrigger.tsx:260]: Change to

> Send Email

File: components/Paybutton/PaybuttonTrigger.tsx [hunk 2]
```diff
@@ -147,110 +151,218 @@ export default ({ paybuttonId, emailCredits }: IProps): JSX.Element => {
...
+                      </button>
+                    )}
+                    <button
+                      disabled={disableEmailSubmit}
+                      type="submit"
+                      className="button_main"
+                    >
+                      {currentEmailTriggerId === undefined
+                        ? 'Create Receive Email'
+                        : 'Update Receive Email'}
```
Comment (Klakurka) [inline components/Paybutton/PaybuttonTrigger.tsx:321]: Same here.

**Review by Klakurka — CHANGES_REQUESTED** [review]

File: components/Paybutton/PaybuttonTrigger.tsx [hunk 3]
```diff
@@ -147,110 +151,218 @@ export default ({ paybuttonId, emailCredits }: IProps): JSX.Element => {
...
+              method="post"
+            >
+              <div>
+                <label htmlFor="postURL">URL</label>
+                <input
+                  {...registerPosterTrigger('postURL')}
+                  type="text"
+                  id="postURL"
+                  name="postURL"
+                  placeholder='https://api.example.com'
```
Comment (Klakurka) [inline components/Paybutton/PaybuttonTrigger.tsx:188]: How about just:

> https://paybutton.org

**Comment by johnkuney** [comment]
For sure, added

**Review by Klakurka — APPROVED** [review]

**Pull request merged**